style.css

```diff
@@ -0,0 +1,845 @@
+
+@media only screen and (max-width: 780px) {
+  @font-face {
+    font-family: Inter;
+    src: url("./Assets/fonts/github/Inter-VariableFont_slnt\,wght.ttf");
+  }
+
+  .Navmenu {
+    display: none;
+  }
+
+  body {
+    margin: 0;
+    font-family: Inter, sans-serif;
+  }
+
+  header {
+    height: 50px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .forms-wrapper {
+    display: none;
+  }
+
+  nav {
+    display: flex;
+    width: 95%;
+    height: 50px;
+    margin-left: auto;
+    margin-right: auto;
+    align-items: center;
+    justify-content: space-between;
+    background-color: white;
+    position: fixed;
+  }
+
+  hr {
+    display: none;
+  }
+
+  .nav-h3 {
+    margin-left: 15px;
+    text-decoration: none;
+    color: #172b4d;
+  }
+
+  .intro-wrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 40px;
+    background-image: url("./Assets/Images/header-illsutration-mobile@2x.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .intro-wrappers {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    color: #172b4d;
+    font-size: 40px;
+    line-height: 40px;
+    font-weight: 700px;
+    margin: 0;
+  }
+
+  .intro-wrappers1 {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    color: #36b37f;
+    font-size: 40px;
+    margin: 0;
+  }
+
+  footer {
+    display: none;
+  }
+
+  .intro-wrapper-p {
+    width: 327px;
+    height: 164px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #172b4d;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .intro-wrapper-ul {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    list-style: none;
+    background-color: transparent;
+    width: 100%;
+    padding: 0;
+  }
+
+  .intro-wrapper-ul-li {
+    margin: 5px 10px;
+  }
+
+  .works {
```

Comment:
Kindly note to meet this project milestone requirements, i suggest you use `display:grid` for your card items.
The grid should style cards both in mobile version and desktop versions.
<img width="1680" alt="Screenshot 2022-09-30 at 04 41 47" src="https://user-images.githubusercontent.com/48546992/193262345-e4dd413d-ac01-43e9-a00f-0ffab43abc48.png">